Source/WebCore/page/FocusController.cpp

```diff
@@ -439,16 +439,17 @@ FocusController::FocusController(Page& page, OptionSet<ActivityState> activitySt
 {
 }
 
-void FocusController::setFocusedFrame(LocalFrame* frame)
+void FocusController::setFocusedFrame(Frame* frame)
 {
     ASSERT(!frame || frame->page() == &m_page);
+    m_page.chrome().focusedFrameChanged(frame);
     if (m_focusedFrame == frame || m_isChangingFocusedFrame)
```

Comment:
I don't think I understand why this doesn't work with site isolation.  Could you clarify more in the commit log?